torchdata/dataloader2/reading_service.py

```diff
@@ -187,14 +137,22 @@ class PrototypeMultiProcessingReadingService(ReadingServiceInterface):
             of each epoch in each worker process with ``WorkerInfo``
             and ``DataPipe`` as the expected arguments.
 
+    Note:
+        - This ``ReadingService`` is still in prototype mode and will replace
```

Comment:
unrelated: we should start to graduate it from "prototype mode" and starts find initial pioneering use case adoption~

Reply:
Yea, let me open an issue to track this